### ISSUE TITLE
Directory upload on IE

### ISSUE THREAD

**Issue by regvina** [23/resumable.js#314]
I'm aware that assignBrowse(domNodes, isDirectory) works only in Chrome. Are there any plans of supporting it on IE?
Thanks!

**Comment by steffentchr** [comment]
Is it possible to do this well on IE?

**Comment by thewilli** [comment]
> Is it possible to do this well on IE?

no, it isn't. This would require ActiveX 😜 . Blame MS for (still) ignoring web standards since they won the Browser Wars...
